React/Base/Surface/SurfaceHostingView/RCTSurfaceHostingProxyRootView.h

```diff
@@ -36,6 +36,9 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign) NSTimeInterval loadingViewFadeDelay;
 @property (nonatomic, assign) NSTimeInterval loadingViewFadeDuration;
 
+- (instancetype)initWithSurface:(RCTSurface *)surface
+                sizeMeasureMode:(RCTSurfaceSizeMeasureMode)sizeMeasureMode NS_UNAVAILABLE;
```

Comment:
@jdeff Sorry, I had to remove this one before merging because it caused issues with Facebook's internal builds.

Reply:
@appden Can you share details about the build failure?